app/controllers/userController.ts

```diff
@@ -0,0 +1,66 @@
+import { UserService } from "../services/UserService";
+import User from "../models/user";
+import mongoose from "mongoose";
+import { AppResponse } from "../services/app-responses";
+
+export class UserController {
+  public userService: UserService = new UserService();
+  public response: AppResponse = new AppResponse();
+
+  public getAllUsers = async (req, res, next) => {
+    try {
+      const getAllUsers = await this.userService.getAllUsers();
+      return this.response.successOk(res, "", getAllUsers);
+    } catch (error) {
+      return this.response.errorOnServer(res, "Internal Server Error", error);
+    }
+  };
+
+  public createUser = async (req, res, next) => {
+    try {
+      const userObject = new User({
```

Comment:
@Suresh-ashwini No need to create this object and intiate User ..Just pass req.body to the model in repository

Reply:
New Object creation removed

Reply:
Object creation removed